### ISSUE TITLE
Error: Please pass the correct sensor (races)

### ISSUE THREAD

**Issue by Santanachia** [marcokreeft87/formulaone-card#33]
**Describe the bug**
after adding the card, an error appears

**Expected behavior**
if `state_attr('sensor.formula_one_sensor_races', 'next_race') == None` show information that there is no information, not an error that there is no sensor

**Screenshots**
![image](https://user-images.githubusercontent.com/4628507/208614495-fb3b075c-f8ec-47df-9d43-3b0f2cd45c01.png)

**Additional context**
```yaml
type: custom:formulaone-card
card_type: next_race
sensor: sensor.formula_one_sensor_races
title: Next Race
date_locale: pl
image_clickable: false
```

**Comment by marcokreeft87** [comment]
There is a problem with your sensor. Not the card. Please check if the sensor has data

**Comment by Santanachia** [comment]
it is a correct sensor
![image](https://user-images.githubusercontent.com/4628507/208617992-406fd78c-367e-4940-b8e8-6f2aca63bbd0.png)
But it is empty because: "Formula One API will be down for the remaining of the season..."
https://github.com/delzear/hass-formulaoneapi/blob/master/README.md

The card shows an incorrect(?) error message.

**Comment by marcokreeft87** [comment]
As you can see on : https://github.com/marcokreeft87/formulaone-card/blob/main/src/cards/next-race.ts#L38
The card should show "Season is over. See you next year!" as it does in mine. 

![image](https://user-images.githubusercontent.com/10223677/208620948-dce835a4-c388-4693-8f86-17bc83777ecd.png)

Did you just install the sensor and card integrations?

**Comment by Santanachia** [comment]
I installed via HACS:
https://github.com/delzear/hass-formulaoneapi
https://github.com/marcokreeft87/formulaone-card
I added in `configuration.yaml`:
```yaml
sensor:
  - platform: formulaone_api
  ```
I restarted HA, 4 new sensors appeared, each of them was `unknown`.
I added the card.

It seems to me that the card works correctly when the sensor returns correct data, but it lacks error handling when the sensor works but did not return correct data (because, for example, the integration did not connect to the server).
